// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitBindingPatterns.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [declarationEmitBindingPatterns.d.ts.map]
-{"version":3,"file":"declarationEmitBindingPatterns.d.ts","sourceRoot":"","sources":["declarationEmitBindingPatterns.ts"],"names":[],"mappings":"AAAA,QAAA,MAAM,CAAC,aAAkB;IACjB,CAAC,CAAC,EAAE,MAAM,CAAC;CACd,KAAG,IAAW,CAAA;AAEnB,QAAA,IAAI,CAAC,EAAE,GAAG,CAAC;AACX,iBAAS,CAAC,CAAC,EAAE,GAAE,GAAO,EAAE,EAAE,GAAE,GAAO,EAAE,EAAE,CAAC,EAAE,EAAM,EAAC,GAAE,GAAO,GAAG,IAAI,CAChE"}
+{"version":3,"file":"declarationEmitBindingPatterns.d.ts","sourceRoot":"","sources":["declarationEmitBindingPatterns.ts"],"names":[],"mappings":"AAAA,QAAA,MAAM,CAAC,GAAI,EAAC,CAAC,EAAE,CAAO,EAAC,EAAE;IACjB,CAAC,CAAC,EAAE,MAAM,CAAC;CACd,KAAG,IAAW,CAAA;AAEnB,QAAA,IAAI,CAAC,EAAE,GAAG,CAAC;AACX,iBAAS,CAAC,CAAC,EAAE,GAAE,GAAO,EAAE,EAAE,GAAE,GAAO,EAAE,EAAE,CAAC,EAAE,EAAM,EAAC,GAAE,GAAO,GAAG,IAAI,CAChE"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBrOiAoeyB4OiB6IH06IHsNCiAgICB4Pzogc3RyaW5nOw0KfSkgPT4gdm9pZDsNCmRlY2xhcmUgdmFyIGE6IGFueTsNCmRlY2xhcmUgZnVuY3Rpb24gZih7fT86IGFueSwgW10/OiBhbnksIHsgcDoge30gfT86IGFueSk6IHZvaWQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1kZWNsYXJhdGlvbkVtaXRCaW5kaW5nUGF0dGVybnMuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0QmluZGluZ1BhdHRlcm5zLmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJkZWNsYXJhdGlvbkVtaXRCaW5kaW5nUGF0dGVybnMudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsUUFBQSxNQUFNLENBQUMsYUFBa0I7SUFDakIsQ0FBQyxDQUFDLEVBQUUsTUFBTSxDQUFDO0NBQ2QsS0FBRyxJQUFXLENBQUE7QUFFbkIsUUFBQSxJQUFJLENBQUMsRUFBRSxHQUFHLENBQUM7QUFDWCxpQkFBUyxDQUFDLENBQUMsRUFBRSxHQUFFLEdBQU8sRUFBRSxFQUFFLEdBQUUsR0FBTyxFQUFFLEVBQUUsQ0FBQyxFQUFFLEVBQU0sRUFBQyxHQUFFLEdBQU8sR0FBRyxJQUFJLENBQ2hFIn0=,Y29uc3QgayA9ICh7eDogeiA9ICd5J306IHsKICAgICAgICB4Pzogc3RyaW5nOwogICAgfSk6IHZvaWQgPT4geyB9Cgp2YXIgYTogYW55OwpmdW5jdGlvbiBmKHt9OiBhbnkgPSBhLCBbXTogYW55ID0gYSwgeyBwOiB7fSA9IGF9OiBhbnkgPSBhKTogdm9pZCB7Cn0=
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBrOiAoeyB4OiB6IH06IHsNCiAgICB4Pzogc3RyaW5nOw0KfSkgPT4gdm9pZDsNCmRlY2xhcmUgdmFyIGE6IGFueTsNCmRlY2xhcmUgZnVuY3Rpb24gZih7fT86IGFueSwgW10/OiBhbnksIHsgcDoge30gfT86IGFueSk6IHZvaWQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1kZWNsYXJhdGlvbkVtaXRCaW5kaW5nUGF0dGVybnMuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0QmluZGluZ1BhdHRlcm5zLmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJkZWNsYXJhdGlvbkVtaXRCaW5kaW5nUGF0dGVybnMudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsUUFBQSxNQUFNLENBQUMsR0FBSSxFQUFDLENBQUMsRUFBRSxDQUFPLEVBQUMsRUFBRTtJQUNqQixDQUFDLENBQUMsRUFBRSxNQUFNLENBQUM7Q0FDZCxLQUFHLElBQVcsQ0FBQTtBQUVuQixRQUFBLElBQUksQ0FBQyxFQUFFLEdBQUcsQ0FBQztBQUNYLGlCQUFTLENBQUMsQ0FBQyxFQUFFLEdBQUUsR0FBTyxFQUFFLEVBQUUsR0FBRSxHQUFPLEVBQUUsRUFBRSxDQUFDLEVBQUUsRUFBTSxFQUFDLEdBQUUsR0FBTyxHQUFHLElBQUksQ0FDaEUifQ==,Y29uc3QgayA9ICh7eDogeiA9ICd5J306IHsKICAgICAgICB4Pzogc3RyaW5nOwogICAgfSk6IHZvaWQgPT4geyB9Cgp2YXIgYTogYW55OwpmdW5jdGlvbiBmKHt9OiBhbnkgPSBhLCBbXTogYW55ID0gYSwgeyBwOiB7fSA9IGF9OiBhbnkgPSBhKTogdm9pZCB7Cn0=
 
